src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -86,6 +101,11 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit, AfterViewIni
     return this._isExpanded;
   }
 
+  /**
+   * Indicates whether the repeater item's checkbox is selected.
+   * When users select the repeater item, the specified property on your model is updated accordingly. This property accepts boolean values.
```

Comment:
We need to delete `This property accepts boolean values.`

Reply:
Oops. Good catch. Done.